### PR TITLE
Strings should not be concatenated using '+' in a loop

### PR DESCRIPTION
**Pull request by bitchelov**
### What is this PR for?
Further, performance of these intermediate operations degrades as the String gets longer. Therefore, the use of StringBuilder is preferred.

### What type of PR is it?
[Refactoring]

### Todos
* [ ] - Task

### What is the Jira issue?
* https://issues.apache.org/jira/browse/ZEPPELIN-1839

### How should this be tested?
InputTest.java (still working)

### Questions:
* Does the licenses files need update? (no)
* Is there breaking changes for older versions? (no)
* Does this needs documentation? (no)

### CODE REVIEW

**Comment by AhyoungRyu** [comment]
@bitchelov Did you have a chance to look into [this comment](https://issues.apache.org/jira/browse/ZEPPELIN-1839?focusedCommentId=15761118&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-15761118) by @zjffdu ? I think it would be better not only the PR description but also the  related Jira issue has more precise description what needs to be improved :)

**Comment by bitchelov** [comment]
@Leemoonsoo https://issues.apache.org/jira/secure/ViewProfile.jspa?name=Sergey+Sokur

**Comment by bitchelov** [comment]
@AhyoungRyu ok, ill update jira issue.

**Comment by Leemoonsoo** [comment]
Thanks @bitchelov LGTM and merge to master if there're no further comments.

File: zeppelin-interpreter/src/main/java/org/apache/zeppelin/display/Input.java [hunk 1]
```diff
@@ -108,7 +108,7 @@ public Input(String name, Object defaultValue, String type, ParamOption[] option
   }
 
   public Input(String name, String displayName, String type, String argument, Object defaultValue,
-      ParamOption[] options, boolean hidden) {
+               ParamOption[] options, boolean hidden) {
```
Comment (jongyoul) [inline zeppelin-interpreter/src/main/java/org/apache/zeppelin/display/Input.java]: This project don't allow vertical alignment basically. Please fix it.

Comment (bitchelov) [inline zeppelin-interpreter/src/main/java/org/apache/zeppelin/display/Input.java]: @jongyoul, what do you mean? remove the indent, which added by idea?

Comment (jongyoul) [inline zeppelin-interpreter/src/main/java/org/apache/zeppelin/display/Input.java]: @bitchelov Apache Zeppelin, basically, follows Google Java style guide. You can check it: https://google.github.io/styleguide/javaguide.html#s4.6.3-horizontal-alignment

**Comment by jongyoul** [comment]
LGTM. will merge it

**Pull request closed without merging**